cognitedata/reveal
# feat: add pointer event in 360 annotation interaction events

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-924

## Description :pencil:


Add pointer event data to the annotation interaction events.

## How has this been tested? :mag:



- Open 360 image (e.g. `XOM-HCU1_north_v02` in 3d-test, find annotations as shown in https://github.com/cognitedata/reveal/pull/3201) 
- Clickyclick
- Observe that pointer event data is printed to console

## Test instructions :information_source:




## Checklist :ballot_box_with_check:
